src/utils/BucketUtils.js

```diff
@@ -16,10 +16,12 @@ import {
   pick,
   toLower,
   uniq,
-  values
+  values,
 } from 'lodash/fp';
-import { Match } from '../libs/ajax/Match';
-import {translateDataUseRestrictionsFromDataUseArray} from '../libs/dataUseTranslation';
+import {Match} from '../libs/ajax/Match';
+import {
+  translateDataUseRestrictionsFromDataUseArray,
+} from '../libs/dataUseTranslation';
```

Comment:
Can we standardize on a format, to denoise PRs across author?  Let me know if this seems worthwhile, and I can open a tech health ticket.

Reply:
Looked into this a bit and we definitely will want to do this in an independent ticket so that we can both update the eslint.config.js with the formatting rule and then update all of the files to the standard.